packages/zend-form/library/Zend/Form/Element/Captcha.php

```diff
@@ -201,7 +201,7 @@ public function render(Zend_View_Interface $view = null)
      */
     public function getPluginLoader($type)
     {
-        $type = strtoupper($type);
+        $type = is_string($type) ? strtoupper($type) : $type;
```

Comment:
idem

Reply:
I will leave this one as it calls `parent::getPluginLoader($type)` and theoretically that could be called with `null` and handled there.